---
title: "RNA-Seq Master Guide"
subtitle: "Version 1.1"
author: 
  - name: "Sasha Thalluri, Lasseigne Lab, CDIB, UAB"
  - name: "Developed with assistance from the Lasseigne Lab and Dr. Lara Ianov, UAB"
    affiliation: For questions, email Dr. Lasseigne at bnp0001@uab.edu
date: "4/2/2020 Last Updated: 5/26/2020"
output:
  html_document:
    theme: spacelab
    toc: yes
    toc_float:
      collapsed: no
      smooth_control: yes
  pdf_document:
    toc: yes
---

# Introduction into RNA-Seq
***
#### This is an SOP on getting started with RNA-seq. Here you can find information on resources, aligners, DESeq2, and pathway analysis tools. 
<br>

# Resources
***

This is a list of links to resources that have essential information on RNA-seq and the recommended R packages for RNA-seq analysis. 

* <a href="https://www.youtube.com/watch?v=YzkTc6VNNjo&feature=youtu.be">Introduction to RNA Sequencing</a>
  * Background lecture for RNA-seq
  * <a href="https://drive.google.com/file/d/1Ldp6qrAMy0NVCCAu5cueT4YFq0VArORH/view">Here</a> are the slides used in the lecture
  
* <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4863231/">RNA Sequencing and Analysis</a>
  * Paper giving an overall introduction to RNA-seq

* <a href="https://peerj.com/preprints/27283/">RNA sequencing data: hitchhiker's guide to expression analysis</a>
  * Paper stating the developments in RNA-seq data analysis 
  * Focuses on quantification of gene expression and statistical approaches for differential expression

* <a href="https://www.nature.com/articles/s41576-019-0150-2">RNA sequencing: the teenage years</a>
  * Paper stating the different RNA-seq methods, while naming certain RNA technologies and good computational tools for data analysis 
  
# About Aligners 
***
<p class="text-muted">Contributed by Dr. Lara Ianov, UAB</p>

When RNA-seq alignment and mapping are discussed, there are generally two primary methods of alignment: 1) **standard splice-aware alignment to the genome** (e.g.: STAR, HISAT2), **and quasi-mapping/pseudoalignment approaches** which utilize k-mer based counting methods to map fragments to the transcriptome (e.g.: Salmon, kallisto). STAR <a href="https://academic.oup.com/bioinformatics/article/29/1/15/272537">(Dobin et al., 2013)</a> is usually my choice for a splice-aware aligner and Salmon <a href="https://pubmed.ncbi.nlm.nih.gov/28263959/">(Patro, Duggal, Love, Irizarry, & Kingsford, 2017)</a> for quasi-mapping. *The choice between the two generally depend on the specific long-term goal of a study.* It is worth noting that there are significant differences in the computational efficiency among these methods, but this goes beyond the scope of this synopsis.

While some benchmarking studies or computational community-based benchmarking have pointed to increased performance from quasi-mapping/pseudoalignment approaches, others have indicated that all modern methods perform similarly for mRNAs and highly abundant genes <a href="https://cgatoxford.wordpress.com/2016/08/17/why-you-should-stop-using-featurecounts-htseq-or-cufflinks2-and-start-using-kallisto-salmon-or-sailfish/">(Smith, 2016;</a> <a href="https://link.springer.com/article/10.1186/s12864-018-4869-5">Wu, Yao, Ho, Lambowitz, & Wilke, 2018)</a>. Thus, one may conclude that if the goal of the study is to perform standard mRNA-seq, any differences found among the methods will not results in a large loss of information if any.

However, beyond standard gene-level mRNA-seq quantification, here is an overview of some of the key points where choosing one approach over another may have a large effect in the interpretation of the results:

* For long non-coding RNAs quantification, quasi-mapping/pseudoalignment have consistently outperformed standard splice-aware alignment methods <a href="https://academic.oup.com/gigascience/article/8/12/giz145/5663671">(Zheng, Brennan, Hernaez, & Gevaert, 2019)</a>.
* On the other hand, it has been reported that standard splice-aware RNA-seq aligners are more sensitive towards small RNAs (including miRNAs) or low abundant genes <a href="https://link.springer.com/article/10.1186/s12864-018-4869-5">(Wu et al., 2018)</a>.
* Gene-level vs transcript-levels quantification: it is important to keep in mind that in most cases standard slice-aware aligners and down-stream quantification methods are meant to target gene-level analysis as there are challenges with multi-mapping reads for transcript variant quantification. Transcript-level analysis is a much-debated topic, but the use of quasi-mapping coupled with novel down-stream quantification tools <a href="https://www.biorxiv.org/content/10.1101/2020.04.07.029967v1.abstract">(Sarkar, Srivastava, Bravo, Love, & Patro, 2020)</a> for transcript-level analysis are more sensitive <a href="https://cgatoxford.wordpress.com/2016/08/17/why-you-should-stop-using-featurecounts-htseq-or-cufflinks2-and-start-using-kallisto-salmon-or-sailfish/">(Smith, 2016)</a>. 
  * However, if the goal of the analysis is to investigate splicing events (e.g.: classifying what type of splicing event occurred) or fusion events rather than transcript abundance quantification, most of the tools developed currently depend on outputs from standard splice-aware RNA-seq aligners (although this may change in the near future as tools such as Terminus cited above finish development/are updated).
* If variant analysis is to be performed on RNA-seq, then splice-aware alignment methods must be used. It could also be argued that if RNA-seq data will be closely compared to whole-genome sequencing dataset, then it may be best to use the slice-aware aligner methods as well (especially if genome browsers will be used heavily).
* For diagnostic goals (e.g.: rare disease cases), most tools available have a dependency on standard alignment approaches as well.

# DESeq2
***
DESeq2 is an R package used for differential analysis on count data from high-throughput sequencing assays. It allows for quantitative analysis focused on strength rather than the mere presence of differential expression. 

**The required input for DESeq2 is gene or transcript counts.**  

<a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-014-0550-8">Here</a> is the paper that will give an overall introduction into DESeq2. It explains how DESeq2 uses shrinkage estimation for dispersions and fold changes to improve stability and interpretability of estimates. 

Instructions on how to install DESeq2 into RStudio can be found <a href="http://bioconductor.org/packages/release/bioc/html/DESeq2.html">here.</a> 

After installing DESeq2, you can begin the vignette. The link to the vignette can be found <a href="http://bioconductor.org/packages/release/bioc/vignettes/DESeq2/inst/doc/DESeq2.html#standard-workflow">here.</a> 

<a href="http://master.bioconductor.org/packages/release/workflows/vignettes/rnaseqGene/inst/doc/rnaseqGene.html">Here</a> is Mike Love's guide for DESeq2. This is an additional resource that explains how to use the package and demonstrates useful workflows.

Please be aware there are many other approaches for estimating differential expression that may be appropriate, or more appropriate (e.g., time course analysis), for your project. We recommend discussing with a knowledgeable data scientist if you are unsure if DESeq2 is a good option for your experimental design.

# Pathway Analysis Tools for R
***
Below are links to vignettes/tutorials for R packages that are helpful for pathway analysis. There are many other tools and analytical approaches and we intend to add to this section in the future.

**gprofiler2**

The gprofiler2 package provides a tool set for functional enrichment analysis and visualization. 
It is primarily used to visualize gene lists, convert gene/protein/SNP identifiers to numerous namespaces, and map orthologous genes across species. <a href="https://www.nature.com/articles/s41596-018-0103-9">Here</a> is the paper that will give an overall introduction to pathway analysis using the gprofiler2 package. 

<a href="https://cran.r-project.org/web/packages/gprofiler2/vignettes/gprofiler2.html">Here</a> is the link to the vignette, which will give you clear instructions on how to use the package. 

**GAGE**

Generally Applicable Gene-set Enrichment (GAGE) is a method for gene set or pathway analysis. The gage package can be used on microarray or RNA-seq data for routine and advanced gene set analyses. <a href="https://bioconductor.org/packages/release/bioc/html/gage.html">Here</a> is the link to the Bioconductor webpage, which will background information on the package and instructions on how to install it into RStudio. 

<a href="https://bioconductor.org/packages/release/bioc/vignettes/gage/inst/doc/gage.pdf">Here</a> is the link to the vignette. 
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<center> Version 1.1: Added RNA-Seq alignment information from Dr. Ianov </center>
<br>

