---
title: "RNA-Seq Master Guide"
subtitle: "Version 1.1"
author: 
  - name: "Sasha Thalluri, Lasseigne Lab, CDIB, UAB"
  - name: "Developed with assistance from the Lasseigne Lab and Dr. Lara Ianov, UAB"
    affiliation: For questions, email Dr. Lasseigne at bnp0001@uab.edu
date: "4/2/2020 Last Updated: 5/26/2020"
output:
  html_document:
    theme: spacelab
    toc: yes
    toc_float:
      collapsed: no
      smooth_control: yes
  pdf_document:
    toc: yes
---



<div id="introduction-into-rna-seq" class="section level1">
<h1>Introduction into RNA-Seq</h1>
<hr />
<div id="this-is-an-sop-on-getting-started-with-rna-seq.-here-you-can-find-information-on-resources-aligners-deseq2-and-pathway-analysis-tools." class="section level4">
<h4>This is an SOP on getting started with RNA-seq. Here you can find information on resources, aligners, DESeq2, and pathway analysis tools.</h4>
<p><br></p>
</div>
</div>
<div id="resources" class="section level1">
<h1>Resources</h1>
<hr />
<p>This is a list of links to resources that have essential information on RNA-seq and the recommended R packages for RNA-seq analysis.</p>
<ul>
<li><a href="https://www.youtube.com/watch?v=YzkTc6VNNjo&feature=youtu.be">Introduction to RNA Sequencing</a>
<ul>
<li>Background lecture for RNA-seq</li>
<li><a href="https://drive.google.com/file/d/1Ldp6qrAMy0NVCCAu5cueT4YFq0VArORH/view">Here</a> are the slides used in the lecture</li>
</ul></li>
<li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4863231/">RNA Sequencing and Analysis</a>
<ul>
<li>Paper giving an overall introduction to RNA-seq</li>
</ul></li>
<li><a href="https://peerj.com/preprints/27283/">RNA sequencing data: hitchhiker’s guide to expression analysis</a>
<ul>
<li>Paper stating the developments in RNA-seq data analysis</li>
<li>Focuses on quantification of gene expression and statistical approaches for differential expression</li>
</ul></li>
<li><a href="https://www.nature.com/articles/s41576-019-0150-2">RNA sequencing: the teenage years</a>
<ul>
<li>Paper stating the different RNA-seq methods, while naming certain RNA technologies and good computational tools for data analysis</li>
</ul></li>
</ul>
</div>
<div id="about-aligners" class="section level1">
<h1>About Aligners</h1>
<hr />
<p class="text-muted">
Contributed by Dr. Lara Ianov, UAB
</p>
<p>When RNA-seq alignment and mapping are discussed, there are generally two primary methods of alignment: <strong>standard splice-aware alignment to the genome</strong> (e.g.: STAR, HISAT2) and <strong>quasi-mapping/pseudoalignment approaches</strong>, which utilize k-mer based counting methods to map fragments to the transcriptome (e.g.: Salmon, kallisto). STAR <a href="https://academic.oup.com/bioinformatics/article/29/1/15/272537">(Dobin et al., 2013)</a> is usually my choice for a splice-aware aligner and Salmon <a href="https://pubmed.ncbi.nlm.nih.gov/28263959/">(Patro, Duggal, Love, Irizarry, &amp; Kingsford, 2017)</a> for quasi-mapping. <em>The choice between the two generally depend on the specific long-term goal of a study.</em> It is worth noting that there are significant differences in the computational efficiency among these methods, but this goes beyond the scope of this synopsis.</p>
<p>While some benchmarking studies or computational community-based benchmarking have pointed to increased performance from quasi-mapping/pseudoalignment approaches, others have indicated that all modern methods perform similarly for mRNAs and highly abundant genes <a href="https://cgatoxford.wordpress.com/2016/08/17/why-you-should-stop-using-featurecounts-htseq-or-cufflinks2-and-start-using-kallisto-salmon-or-sailfish/">(Smith, 2016;</a> <a href="https://link.springer.com/article/10.1186/s12864-018-4869-5">Wu, Yao, Ho, Lambowitz, &amp; Wilke, 2018)</a>. Thus, one may conclude that if the goal of the study is to perform standard mRNA-seq, any differences found among the methods will not result in a large loss of information if any.</p>
<p>However, beyond standard gene-level mRNA-seq quantification, here is an overview of some of the key points where choosing one approach over another may have a large effect in the interpretation of the results:</p>
<ul>
<li>For long non-coding RNAs quantification, quasi-mapping/pseudoalignment have consistently outperformed standard splice-aware alignment methods <a href="https://academic.oup.com/gigascience/article/8/12/giz145/5663671">(Zheng, Brennan, Hernaez, &amp; Gevaert, 2019)</a>.</li>
<li>On the other hand, it has been reported that standard splice-aware RNA-seq aligners are more sensitive towards small RNAs (including miRNAs) or low abundant genes <a href="https://link.springer.com/article/10.1186/s12864-018-4869-5">(Wu et al., 2018)</a>.</li>
<li>Gene-level vs transcript-levels quantification: it is important to keep in mind that in most cases standard splice-aware aligners and down-stream quantification methods are meant to target gene-level analysis as there are challenges with multi-mapping reads for transcript variant quantification. Transcript-level analysis is a much-debated topic, but the use of quasi-mapping coupled with novel down-stream quantification tools <a href="https://www.biorxiv.org/content/10.1101/2020.04.07.029967v1.abstract">(Sarkar, Srivastava, Bravo, Love, &amp; Patro, 2020)</a> for transcript-level analysis are more sensitive <a href="https://cgatoxford.wordpress.com/2016/08/17/why-you-should-stop-using-featurecounts-htseq-or-cufflinks2-and-start-using-kallisto-salmon-or-sailfish/">(Smith, 2016)</a>.
<ul>
<li>However, if the goal of the analysis is to investigate splicing events (e.g.: classifying what type of splicing event occurred) or fusion events rather than transcript abundance quantification, most of the tools developed currently depend on outputs from standard splice-aware RNA-seq aligners (although this may change in the near future as tools such as Terminus cited above finish development/are updated).</li>
</ul></li>
<li>If variant analysis is to be performed on RNA-seq, then splice-aware alignment methods must be used. It could also be argued that if RNA-seq data will be closely compared to whole-genome sequencing dataset, then it may be best to use the slice-aware aligner methods as well (especially if genome browsers will be used heavily).</li>
<li>For diagnostic goals (e.g.: rare disease cases), most tools available have a dependency on standard alignment approaches as well.</li>
</ul>
</div>
<div id="deseq2" class="section level1">
<h1>DESeq2</h1>
<hr />
<p>DESeq2 is an R package used for differential analysis on count data from high-throughput sequencing assays. It allows for quantitative analysis focused on strength rather than the mere presence of differential expression.</p>
<p><strong>The required input for DESeq2 is gene or transcript counts.</strong></p>
<p><a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-014-0550-8">Here</a> is the paper that will give an overall introduction into DESeq2. It explains how DESeq2 uses shrinkage estimation for dispersions and fold changes to improve stability and interpretability of estimates.</p>
<p>Instructions on how to install DESeq2 into RStudio can be found <a href="http://bioconductor.org/packages/release/bioc/html/DESeq2.html">here.</a></p>
<p>After installing DESeq2, you can begin the vignette. The link to the vignette can be found <a href="http://bioconductor.org/packages/release/bioc/vignettes/DESeq2/inst/doc/DESeq2.html#standard-workflow">here.</a></p>
<p><a href="http://master.bioconductor.org/packages/release/workflows/vignettes/rnaseqGene/inst/doc/rnaseqGene.html">Here</a> is Mike Love’s guide for DESeq2. This is an additional resource that explains how to use the package and demonstrates useful workflows.</p>
<p>Please be aware there are many other approaches for estimating differential expression that may be appropriate, or more appropriate (e.g., time course analysis), for your project. We recommend discussing with a knowledgeable data scientist if you are unsure if DESeq2 is a good option for your experimental design.</p>
</div>
<div id="pathway-analysis-tools-for-r" class="section level1">
<h1>Pathway Analysis Tools for R</h1>
<hr />
<p>Below are links to vignettes/tutorials for R packages that are helpful for pathway analysis. There are many other tools and analytical approaches and we intend to add to this section in the future.</p>
<p><strong>gprofiler2</strong></p>
<p>The gprofiler2 package provides a tool set for functional enrichment analysis and visualization.
It is primarily used to visualize gene lists, convert gene/protein/SNP identifiers to numerous namespaces, and map orthologous genes across species. <a href="https://www.nature.com/articles/s41596-018-0103-9">Here</a> is the paper that will give an overall introduction to pathway analysis using the gprofiler2 package.</p>
<p><a href="https://cran.r-project.org/web/packages/gprofiler2/vignettes/gprofiler2.html">Here</a> is the link to the vignette, which will give you clear instructions on how to use the package.</p>
<p><strong>GAGE</strong></p>
<p>Generally Applicable Gene-set Enrichment (GAGE) is a method for gene set or pathway analysis. The gage package can be used on microarray or RNA-seq data for routine and advanced gene set analyses. <a href="https://bioconductor.org/packages/release/bioc/html/gage.html">Here</a> is the link to the Bioconductor webpage, which will background information on the package and instructions on how to install it into RStudio.</p>
<a href="https://bioconductor.org/packages/release/bioc/vignettes/gage/inst/doc/gage.pdf">Here</a> is the link to the vignette.
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<center>
Version 1.1: Added RNA-Seq alignment information from Dr. Ianov
</center>
<p><br></p>
</div>
