---
title: "Single-Cell Transcriptomics Guide"
subtitle: "Version 1.1.1"
author: 
  - name: "Sasha Thalluri, Lasseigne Lab, CDIB, UAB"
  - name: "Developed with assistance from the Lasseigne Lab and Dr. Lara Ianov, UAB"
    affiliation: For questions, email Dr. Lasseigne at bnp0001@uab.edu
date: "5/5/2020"
output:
  html_document:
    theme: spacelab
    toc: yes
    toc_float:
      collapsed: no
      smooth_control: yes
  pdf_document:
    toc: yes
---



<div id="introduction-into-single-cell" class="section level1">
<h1>Introduction into Single-Cell</h1>
<hr />
<div id="this-is-an-sop-on-getting-started-with-single-cell-transcriptomics.-here-you-can-find-papers-courses-databases-and-tools." class="section level4">
<h4>This is an SOP on getting started with Single-Cell Transcriptomics. Here you can find papers, courses, databases, and tools.</h4>
<p><br></p>
</div>
</div>
<div id="papers" class="section level1">
<h1>Papers</h1>
<hr />
<p>This is a list of links to papers that have essential information on single-cell transcriptomics.
<br></p>
<p><a href="https://genomemedicine.biomedcentral.com/articles/10.1186/s13073-017-0467-4">A practical guide to single-cell RNA-sequencing for biomedical research and clinical applications</a></p>
<ul>
<li>Review paper that presents a practical guide to help researchers design their first scRNA-seq studies</li>
<li>Includes introductory information on
<ul>
<li>Experimental hardware</li>
<li>Protocol choice</li>
<li>Quality control</li>
<li>Data analysis</li>
<li>Biological interpretation</li>
</ul></li>
</ul>
<p><a href="https://www.nature.com/articles/s41587-020-00744-z">Guidelines for reporting single-cell RNA-seq experiments</a></p>
<ul>
<li>Article that provides guidelines for the minimum set of metadata needed for robust comparative analyses of scRNA-seq</li>
</ul>
<p><a href="https://www.nature.com/articles/s41576-019-0118-2">Optimizing biological inferences from single-cell data</a></p>
<ul>
<li>Article that discusses the exciting opportunities of single-cell omic studies</li>
<li>Highlights the importance of appropriate data analysis strategies</li>
</ul>
<p><a href="https://www.nature.com/articles/s41587-020-0469-4">Single-cell RNA-sequencing protocols for cell atlas projects</a></p>
<ul>
<li>Paper that reveals the differences between different scRNA-seq and single-nucleus RNA-seq protocols</li>
<li>States that the differing protocols impact the predictive value and suitability for integration into reference cell atlases</li>
<li>Results are particularly helpful for individual researchers and consortium projects</li>
</ul>
<p><a href="https://www.nature.com/articles/s41576-020-0229-9">Transcriptional profiling of physically interacting cells</a></p>
<ul>
<li>Paper highlighting the use of PIC-seq</li>
<li>PIC-seq combines cell sorting of physically interacting cells (PICs) with scRNA-seq and computational modeling to profile cellular interactions and their impact on gene expression</li>
</ul>
<p><a href="https://www.nature.com/articles/s41587-020-0470-y">Combinatorial single-cell CRISPR screens by direct guide RNA capture and targeted sequencing</a></p>
<ul>
<li>Paper demonstrating how Pertub-seq allows for pooled single-cell CRISPR screens to be easily paired with combinatorial perturbation libraries</li>
</ul>
<p><a href="https://www.nature.com/articles/s41576-019-0093-7">Integrative single-cell analysis</a></p>
<ul>
<li>Review paper that discuses the recent advances in the collection and integration of different data types at single-cell resolution</li>
<li>Focuses on the integration of gene expression data with other types of single-cell measurement</li>
</ul>
<p><a href="https://www.nature.com/articles/s41467-019-12266-7">A systematic evaluation of single cell RNA-seq analysis pipelines</a></p>
<ul>
<li>Paper that assesses interactions among pipeline steps</li>
</ul>
<p><a href="https://www.cell.com/cell/fulltext/S0092-8674(20)30346-9">The Human Tumor Atlas Network</a></p>
<ul>
<li>Paper for single-cell analysis in tumors</li>
<li>Focuses on current challenges and future directions</li>
</ul>
<p><a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-018-1603-1">Cell Hashing with barcoded antibodies enables multiplexing and doublet detection for single cell genomics</a></p>
<ul>
<li>Paper that demonstrates how hashing can generalize the benefits of single cell multiplexing to diverse samples and experimental designs</li>
</ul>
<p><a href="https://pair-code.github.io/understanding-umap/">Understanding UMAP</a></p>
<ul>
<li>Review that focuses on dimensionality reduction using a new technique called UMAP</li>
<li>Includes information on
<ul>
<li>How the UMAP algorithm works</li>
<li>How to use UMAP effectively</li>
<li>How UMAP compares with t-SNE (a widely used technique for visualizing)</li>
</ul></li>
</ul>
<p><br></p>
</div>
<div id="courses" class="section level1">
<h1>Courses</h1>
<hr />
<p>These are courses and other resources that will give an overall introduction to single cell analysis and provide opportunities to learn about computational biology strategies using scRNA-seq
<br>
<br>
<strong>Introduction to Single-Cell RNA-Seq Technologies</strong></p>
<p>Here are slides from a class that focuses on the computational biology of single-cell RNA-seq analysis. To access the slides, click <a href="https://liorpachter.wordpress.com/2019/02/19/introduction-to-single-cell-rna-seq-technologies/">here.</a></p>
<p><strong>Analysis of Single Cell RNA-Seq Data</strong></p>
<p>A course that covers all of the steps of scRNA-seq processing. It includes common analysis strategies and discusses central biological questions that can be addressed using scRNA-seq. To access this course, click <a href="https://broadinstitute.github.io/2019_scWorkshop/index.html">here.</a> Be sure to look at the section titled ‘Identifying Cell Populations’. It provides essential information about cell types. To access that section, click <a href="https://broadinstitute.github.io/2019_scWorkshop/identifying-cell-populations.html">here.</a></p>
<p><strong>Analysis of Single Cell RNA-Seq Data (University of Cambridge)</strong></p>
<p>A course that discusses questions that can be addressed using scRNA-seq and the current computational and statistical methods available. It was designed for those who are interested in learning about computational analysis of scRNA-seq data. To access this course, click <a href="https://scrnaseq-course.cog.sanger.ac.uk/website/index.html">here.</a></p>
<p><strong>Overview Talk of Single Cell Genomics Day 2020</strong></p>
<p>Here is an overview of the new computational analysis strategies and experimental technologies presented at Single Cell Genomics Day 2020.
To access this video, click <a href="https://www.youtube.com/watch?v=b9bqKulMLp8">here.</a> To access the slides used in the video, click <a href="https://satijalab.org/scgd/">here.</a></p>
<p><strong>Machine Learning for Single Cell Analysis Workshop</strong></p>
<p>A course that helps its participants receive an introduction to emerging trends in single cell analysis, learn how to analyze single cell data sets, and develop an understanding of machine learning foundations. To access this workshop, click <a href="https://www.krishnaswamylab.org/workshop">here.</a></p>
<p><br></p>
</div>
<div id="databases" class="section level1">
<h1>Databases</h1>
<hr />
<p>Here are databases/portals/atlases that can be helpful for single-cell analysis.
<br>
<br></p>
<p><strong>Broad Institute Single Cell Portal</strong></p>
<p>Single-Cell Portal was developed to facilitate sharing scientific results and disseminating data generated from single cell technologies. To access it, click <a href="https://singlecell.broadinstitute.org/single_cell">here.</a></p>
<p><strong>Mouse Brain Cell Atlas</strong></p>
<p>Mouse Brain Cell Atlas uses the interactive online software (DropViz) to serve as a reference for development, disease, and evolution. To access it, click <a href="http://dropviz.org/">here.</a></p>
<p><strong>CancerSEA</strong>
CancerSEA is a database that aims to comprehensively decode distinct functional states of cancer cells at single-cell resolution. To access it, click <a href="http://biocc.hrbmu.edu.cn/CancerSEA/home.jsp">here.</a></p>
<p><br></p>
</div>
<div id="tools" class="section level1">
<h1>Tools</h1>
<hr />
<p>Here are some R packages and tools that can be used for single-cell analysis.
<br>
<br></p>
<p><strong>Bioconductor Single-Cell R-Packages</strong>
This is a link to a list of R packages for sc-RNA analysis accessible through Bioconductor. To access the list, click <a href="https://bioconductor.org/packages/release/BiocViews.html#___SingleCellData">here.</a></p>
<p><strong>Orchestrating Single-Cell Analysis with Bioconductor</strong></p>
<p>This is a website that teaches some common workflows for scRNA-seq data. It focuses on how to make use of cutting-edge Bioconductor tools to process, analyze, visualize, and explore scRNA-seq data. To access this book, click <a href="https://osca.bioconductor.org">here.</a></p>
<p><strong>Bioconductor workflow for single-cell RNA sequencing: Normalization, dimensionality reduction, clustering, and lineage inference</strong></p>
<p>This is an integrated workflow that provides a step-by-step tutorial to the methodology and associated software for dimensionality reduction, cell clustering, inference of cell lineages and pseudo-times, and differential expression analysis along lineages. To access this workflow, click <a href="https://www.bioconductor.org/help/course-materials/2017/BioC2017/Day2/Workshops/singleCell/doc/workshop.html">here.</a></p>
<p><strong>Introduction to singleCellTK</strong></p>
<p>This is an R package used for interactive scRNA-seq analysis, which allows users to upload raw scRNA-seq count matrices and perform downstream scRNA-seq analysis interactively through a web interface or through R functions using the command line interface. To access this package, click <a href="https://www.bioconductor.org/packages/release/bioc/vignettes/singleCellTK/inst/doc/v01-Introduction_to_singleCellTK.html">here.</a></p>
<p><strong>Stephanie Hicks Lab Projects</strong></p>
<p>This is a list of projects and references from the Stephanie Hicks lab. Here you can find a lot of single-cell work with various goals to understand the depth of single-cell analysis. To access the list of projects, click <a href="https://www.stephaniehicks.com/projects/">here.</a></p>
<p><strong>Multiplexing Cost Calculator</strong></p>
<p>This is a handy single-cell price calculator. To access it, click <a href="https://satijalab.org/costpercell">here.</a></p>
<p><strong>10X Genomics Cell Ranger Pipeline</strong></p>
<p>This a set of analysis pipelines that processes Chromium single-cell RNA-seq output to align reads, generate feature-barcode matrices and perform clustering and gene expression analysis. It utilizes the STAR aligner. To access the pipelines, click <a href="https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/what-is-cell-ranger">here.</a></p>
<p><strong>Drop-Seq Alignment Guide</strong></p>
<p>This has Java tools for analyzing Drop-Seq data. This software pipeline performs many analyses including massive de-multiplexing of the data, alignment of reads to a reference genome, and processing of cellular and molecular barcodes. To access this guide, click <a href="https://github.com/broadinstitute/Drop-seq">here.</a></p>
<p><strong>Seurat R toolkit for single cell genomics</strong></p>
<p>This is an R package designed for quality control, analysis, and exploration of single-cell RNA-seq data. It enables users to identify and interpret sources of heterogeneity from single-cell transcriptomic measurements and to integrate diverse types of single-cell data. To access this package, click <a href="https://satijalab.org/seurat/">here.</a></p>
<p><strong>Alevin</strong></p>
<p>This is a tool used for a quasi-mapping approach. One may consider using Alevin for 10X Genomics or Drop-seq. To access the tool, click <a href="https://salmon.readthedocs.io/en/latest/alevin.html">here.</a></p>
<p><strong>Spatial Alevin</strong></p>
<p>This is a tutorial that will show how to generate the spatially-resolved gene-count matrix for each spot using <strong>Alevin</strong> and visualize it using <strong>Seurat</strong>. To access the tutorial, click <a href="https://combine-lab.github.io/alevin-tutorial/2020/alevin-spatial/">here.</a></p>
<p><strong>ReactomeGSA</strong></p>
<p>This package allows for Gene Set Enrichment Analysis using the Reactome database.
It allows for comparison of independent datasets across species as well as different omics including single-cell RNA-seq analysis. It also has inbuilt plotting tools. To access the paper that introduces this package, click <a href="https://www.biorxiv.org/content/10.1101/2020.04.16.044958v1.full.pdf">here.</a></p>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<center>
<p>Version 1.1.1: Added Füllgrabe, <em>et.al</em>, <a href="https://www.nature.com/articles/s41587-020-00744-z">https://www.nature.com/articles/s41587-020-00744-z</a> and ‘Identifying Cell Populations’ to the Analysis of Single Cell RNA-Seq Data course
<br></p>
</div>
