---
title: "TCGA Guide"
subtitle: "Version 1.0"
author:
  - name: "Avery Williams, Lasseigne Lab, CDIB, UAB"
  - name: "Developed with assistance from the Lasseigne Lab and Dr. Lara Ianov, UAB"
    affiliation: For questions, email Dr. Lasseigne at <a href="mailto:bnp0001@uab.edu" target="_blank">bnp0001@uab.edu</a>
date: "5/29/2020"
output:
  html_document:
    theme: spacelab
    toc: yes
    toc_float:
      collapsed: no
      smooth_control: yes
  pdf_document:
    toc: yes
---

# Introduction to TCGA
The Cancer Genome Atlas (TCGA) is a resource with omics data for over 20,000 cancer patient tissue profiles spanning 33 cancer types. It is a joint effort between the National Cancer Institute and the National Human Genome Research Institute and is publicly available. See the links below for more information:

* Click <a href="https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga" target="_blank">here</a> for the NIH's introduction to TCGA. You can also visit TCGA's Wikipedia page <a href="https://en.wikipedia.org/wiki/The_Cancer_Genome_Atlas" target="_blank">here</a>.
* <a href="https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga/studied-cancers" target="_blank">This</a> page describes cancers studied by the consortium.
* <a href="https://portal.gdc.cancer.gov/" target="_blank">This</a> is the NIH's Genomic Data Commons data portal where you can access their data.
* <a href="https://gdc.cancer.gov/resources-tcga-users/tcga-code-tables/data-types" target="_blank">Here</a> is a list of the data types available in TCGA.

# Literature

Below are some papers you can read for a better understanding of TCGA:

* <a href="https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga/publications" target="_blank">TCGA Research Network Publications</a>
    * a comprehensive list of publications by the TCGA program
* <a href="https://www.nature.com/collections/ffebghgeba/" target="_blank">TCGA Pan-Cancer Analysis (2013)</a> 
  * a collection on the Pan-Cancer initiative's effort to find similarities and differences among the genomic and cellular alterations found in the first dozen tumor types profiled by TCGA. For example:
    * <a href="https://www.nature.com/articles/ng.2764" target="_blank">The Cancer Genome Atlas Pan-Cancer analysis project</a>
      * a commentary article providing further detail to the Pan-Cancer analysis project touched on in the collection summary
    * <a href="https://www.nature.com/articles/ng.2760" target="_blank">Pan-cancer patterns of somatic copy number alteration</a>
      * a paper analyzing somatic copy number alterations' roles in promoting cancer
    * <a href="https://www.nature.com/articles/ng.2762" target="_blank">Emerging landscape of oncogenic signatures across human cancers</a> 
      * a paper on classifying tumors hierarchically based on genetic and epigenetic alterations to show functional event patterns
* <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html" target="_blank">The Pan-Cancer Atlas Collection (2018)</a>
  * a collection of papers  aiming to reclassify human tumor types based on molecular similarity. For example:
    * <a href="https://www.cell.com/cell-reports/fulltext/S2211-1247(18)30436-4?utm_campaign=STMJ_1522958526_SC&utm_channel=WEB&utm_source=WEB&dgcid=STMJ_1522958526_SC" target="_blank">The Cancer Genome Atlas Comprehensive Molecular Characterization of Renal Cell Carcinoma</a>
      * a paper on finding genomic and phenotypic differences associated with different histologic subtypes in the interest of developing subtype-specific therapeutics
    * <a href="https://www.cell.com/cell/fulltext/S0092-8674(18)30313-1?utm_campaign=STMJ_1522958526_SC&utm_channel=WEB&utm_source=WEB&dgcid=STMJ_1522958526_SC" target="_blank">Perspective on Oncogenic Processes at the End of the Beginning of Cancer Genomics</a>
      * a paper on the current state of findings on the molecular processes governing oncogenesis provided in the TCGA PanCancer Atlas
    * <a href="https://www.cell.com/cancer-cell/fulltext/S1535-6108(18)30111-9?utm_campaign=STMJ_1522958526_SC&utm_channel=WEB&utm_source=WEB&dgcid=STMJ_1522958526_SC" target="_blank">Genomic and Functional Approaches to Understanding Cancer Aneuploidy</a>
      * a paper that defines genomic and phenotypic correlates of cancer aneuploidy and provides an experimental approach to study chromosome arm aneuploidy
    * <a href="https://www.cell.com/cell/fulltext/S0092-8674(18)30229-0?utm_campaign=STMJ_1522958526_SC&utm_channel=WEB&utm_source=WEB&dgcid=STMJ_1522958526_SC&code=cell-site" target="_blank">An Integrated TCGA Pan-Cancer Clinical Data Resource to Drive High-Quality Survival Outcome Analytics</a>
      * a paper detailing the TCGA Pan-Cancer Clinical Data Resource, a standardized dataset developed to ensure proper use of TCGA clinical data
* <a href="https://www.nature.com/collections/afdejfafdb" target="_blank">Pan-Cancer Analysis of Whole Genomes (2020) </a>
  * a collection describing a PCAWG Project that performed whole-genome sequencing and integrative analysis on over 2,600 cancers and their matching tissues across 38 tumor types from Nature
* <a href="https://ascopubs.org/doi/pdfdirect/10.1200/CCI.19.00119" target="_blank">Multiomic Integration of Public Oncology Databases in Bioconductor</a>
  * a paper on utilizing TCGA along with cBioPortal to create packages to aid in making fully integrated representations of multiomic pan-cancer databases

  
# Data Analysis
See <a href="https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga/using-tcga/tools" target="_blank">this</a> page for a list of computational tools developed by researchers using TCGA data.

See <a href="https://docs.gdc.cancer.gov/Data_Portal/Users_Guide/Getting_Started/" target="_blank">this</a> page for a user's guide for accessing the GDC data portal.

### TCGAbiolinks
TCGAbiolinks is an R package that can be used for integrative analysis with Genomic Data Commons data. This package facilitates data retrieval and analysis. You can learn more about its aim and install it <a href="https://bioconductor.org/packages/release/bioc/html/TCGAbiolinks.html" target="_blank">here</a>.

After installation, feel free to go through <a href="https://bioconductor.org/packages/release/bioc/vignettes/TCGAbiolinks/inst/doc/index.html" target="_blank">this</a> webpage for a better understanding. There are case studies and workshops available through the page that might also be helpful.

<a href="http://bioconductor.org/packages/release/workflows/vignettes/TCGAWorkflow/inst/doc/TCGAWorkflow.html" target="_blank">This</a> vignette is another walkthrough option for learning the TCGA Workflow.

You can also reference these YouTube videos from the creators of the package:

* <a href="https://youtu.be/eP9C3kKA8eo" target="_blank">TCGAbiolinks: data, analyses & applications by Antonio Colaprico</a>
* <a href="https://youtu.be/YAtikGUR36c" target="_blank">TCGAbiolinks: data, analyses & applications (Network Stability) by Catharina Olsen</a>

### recount2
recount2 is a resource that allows access to thousands of RNA-seq samples from TCGA, GTEx, and the Sequence Read Archive. It can be accessed through the recount2 website and the recount Bioconductor package. See below for some helpful resources:

* <a href="https://bioconductor.org/packages/release/bioc/html/recount.html" target="_blank">Bioconductor page for recount r package</a>
  * This includes a quick start guide to recount2 that can help you get a general idea of the package.
* <a href="https://www.bioconductor.org/packages/devel/workflows/vignettes/recountWorkflow/inst/doc/recount-workflow.html" target="_blank">recount workflow: accessing over 70,000 human RNA-seq samples with Bioconductor</a>
  * This is a more in-depth workflow you can follow.
* <a href="https://jhubiostatistics.shinyapps.io/recount/" target="_blank">recount2: a multi-experiment resource of analysis-ready RNA-seq gene and exon count datasets</a>
  * This is a resource to download the recount2 datasets. It has a user friendly GUI to interact with these datasets, and allows you to choose from popular datasets, GTEx, TCGA, or all of the above.
* <a href="https://www.nature.com/articles/nbt.3838" target="_blank"> Reproducible RNA-seq analysis using _recount2_</a>
  * This is a Nature Biotechnology paper detailing the development and utilization of recount2.
  
<footer> Version 1.1: Added Multiomic Integration paper </footer>