<!doctype html>
<html lang="en">
  <head>
  <meta charset="utf-8">
<title>2022 Preprints - Lasseigne Lab</title>
<meta name="viewport" content="width=device-width, initial-scale=1">


  <link rel="apple-touch-icon" sizes="180x180" href="/favicon/apple-touch-icon.png?v=1">
  <link rel="icon" type="image/png" sizes="32x32" href="/favicon/favicon-32x32.png?v=1">
  <link rel="icon" type="image/png" sizes="16x16" href="/favicon/favicon-16x16.png?v=1">
  <link rel="manifest" href="/favicon/site.webmanifest?v=1">
  
    <link rel="mask-icon" href="/favicon/safari-pinned-tab.svg?v=1" color="#603cba">
    <link rel="shortcut icon" href="/favicon/favicon.ico?v=1">
    <meta name="msapplication-config" content="/favicon/browserconfig.xml?v=1">
  
  <meta name="msapplication-TileColor" content="#603cba">
  <meta name="theme-color" content="#5bbad5">

<meta name="generator" content="Hugo 0.99.1" /><meta itemprop="name" content="2022 Preprints">
<meta itemprop="description" content="CINmetrics: An R package for chromosomal instability analysis
Authors: Vishal H. Orza, Jennifer L. Fisher, Roshan Darjoi, Brittany N. Lasseigne
Genomic instability is an important hallmark of cancer and more recently has been identified in others like neurodegenrative diseases. Chromosomal instability, as a measure of genomic instability, has been used to characterize clinical and biological phenotypes associated with these diseases by measuring structural and numerical chromosomal alterations. There have been multiple chromosomal instability scores developed across many studies in the literature; however, these scores have not been compared because of the lack of a single tool available to calculate and facilitate these various metrics."><meta itemprop="datePublished" content="2023-02-02T00:00:00+00:00" />
<meta itemprop="dateModified" content="2023-02-02T00:00:00+00:00" />
<meta itemprop="wordCount" content="490">
<meta itemprop="keywords" content="lab,website,updates,ISCB," /><meta property="og:title" content="2022 Preprints" />
<meta property="og:description" content="CINmetrics: An R package for chromosomal instability analysis
Authors: Vishal H. Orza, Jennifer L. Fisher, Roshan Darjoi, Brittany N. Lasseigne
Genomic instability is an important hallmark of cancer and more recently has been identified in others like neurodegenrative diseases. Chromosomal instability, as a measure of genomic instability, has been used to characterize clinical and biological phenotypes associated with these diseases by measuring structural and numerical chromosomal alterations. There have been multiple chromosomal instability scores developed across many studies in the literature; however, these scores have not been compared because of the lack of a single tool available to calculate and facilitate these various metrics." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://lasseigne.org/blog/2022-preprints/" /><meta property="article:section" content="blog" />
<meta property="article:published_time" content="2023-02-02T00:00:00+00:00" />
<meta property="article:modified_time" content="2023-02-02T00:00:00+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="2022 Preprints"/>
<meta name="twitter:description" content="CINmetrics: An R package for chromosomal instability analysis
Authors: Vishal H. Orza, Jennifer L. Fisher, Roshan Darjoi, Brittany N. Lasseigne
Genomic instability is an important hallmark of cancer and more recently has been identified in others like neurodegenrative diseases. Chromosomal instability, as a measure of genomic instability, has been used to characterize clinical and biological phenotypes associated with these diseases by measuring structural and numerical chromosomal alterations. There have been multiple chromosomal instability scores developed across many studies in the literature; however, these scores have not been compared because of the lack of a single tool available to calculate and facilitate these various metrics."/>
<meta name="twitter:site" content="@bnlasse"/>
<link rel="stylesheet" href="/css/bundle.min.262e62e9c1615dd1ac95d339cfc4ca2167aee72b71d853838920a22e7088b06b.css" integrity="sha256-Ji5i6cFhXdGsldM5z8TKIWeu5ytx2FODiSCiLnCIsGs=">
        <link rel="stylesheet" href="/css/add-on.css">
</head>

  <body>
    
<header id="site-header">
  <nav id="site-nav">
    <h1 class="nav-title">
      <a href="/">
        
          
            Blog
          
        
      </a>
    </h1>
    <menu id="site-nav-menu" class="flyout-menu">
      
        
          
          
            <a href="/" class="link"><i class='fa fa-home'></i> Home</a>
          
        
      
        
          
          
            <a href="/team/" class="link"><i class='far fa-id-card'></i> Team</a>
          
        
      
        
          
          
            <a href="/research/" class="link"><i class='fas fa-microscope'></i> Research</a>
          
        
      
        
          
          
            <a href="/education/" class="link"><i class='fas fa-chalkboard-teacher'></i> Education</a>
          
        
      
        
          
          
            <a href="/publications/" class="link"><i class='far fa-file-alt'></i> Publications</a>
          
        
      
        
          
          
            <a href="/funding/" class="link"><i class='fas fa-dollar-sign'></i> Funding</a>
          
        
      
        
          
          
            <a href="/contact-us/" class="link"><i class='far fa-envelope'></i> Contact</a>
          
        
      
      
      
    </menu>
    
    
    
    <a href="#site-nav" class="nav-toggle"><i class="fas fa-bars fa-2x"></i></a>
  </nav>
  
  
</header>

    <div id="wrapper">
      <section id="site-intro" >
  <a href="/"><img src="/img/main/lablogo.png" class="circle" width="200" alt="Lasseigne Lab" /></a>
  <header>
    <h1>Lasseigne Lab</h1>
  </header>
  <main>
    <p>Because Science.</p>
  </main>
  
    <footer>
      <ul class="socnet-icons">
        

        <li><a href="//github.com/lasseignelab" target="_blank" rel="noopener" title="GitHub" class="fab fa-github"></a></li>











<li><a href="//linkedin.com/in/brittany-lasseigne-phd" target="_blank" rel="noopener" title="LinkedIn" class="fab fa-linkedin"></a></li>















<li><a href="//twitter.com/bnlasse" target="_blank" rel="noopener" title="Twitter" class="fab fa-twitter"></a></li>







<li><a href="//researchgate.net/profile/Brittany_Lasseigne" target="_blank" rel="noopener" title="Research Gate"><i class="ai ai-researchgate"></i></a></li>


<li><a href="mailto:bnp0001@uab.edu" target="_blank" title="Email" class="far fa-envelope"></a></li>

      </ul>
    </footer>
  
</section>

      <main id="site-main">
        <article class="post">
  <header>
  <div class="title">
    
      <h2><a href="/blog/2022-preprints/">2022 Preprints</a></h2>
    
    
  </div>
</header>

  <section id="socnet-share">
    





  </section>
  

  <div class="content">
    <p><strong><a href="https://www.biorxiv.org/content/10.1101/2021.11.15.467294v2" target="_blank">CINmetrics: An R package for chromosomal instability analysis</a></strong></p>
<p>Authors: Vishal H. Orza, Jennifer L. Fisher, Roshan Darjoi, Brittany N. Lasseigne</p>
<p>Genomic instability is an important hallmark of cancer and more recently has been identified in others like neurodegenrative diseases. Chromosomal instability, as a measure of genomic instability, has been used to characterize clinical and biological phenotypes associated with these diseases by measuring structural and numerical chromosomal alterations. There have been multiple chromosomal instability scores developed across many studies in the literature; however, these scores have not been compared because of the lack of a single tool available to calculate and facilitate these various metrics. Here, we provide an R package CINmetrics, that calculates six different chromosomal instability scores and allows direct comparison between them. We also demonstrate how these scores differ by applying CINmetrics to breast cancer data from The Cancer Genome Atlas (TCGA). The package is available on CRAN at <a href="https://cran.r-project.org/package=CINmetrics">https://cran.r-project.org/package=CINmetrics</a> and on github at <a href="https://github.com/lasseignelab/CINmetrics">https://github.com/lasseignelab/CINmetrics</a>.</p>
<p><strong><a href="https://www.biorxiv.org/content/10.1101/2022.12.02.518863v1" target="_blank">Prioritized polycystic kidney disease drug targets and repurposing candidates from pre-cystic and cystic mouse model gene expression reversion</a></strong></p>
<p>Authors: Elizabeth J. Wilk, Timothy C. Howton, Jennifer L. Fisher, Vishal H. Oza, Ryan T. Brownlee, Kasi C. McPherson, Hannah L. Cleary, Bradley K. Yoder, James F. George, Michal Mrug, Brittany N. Lasseigne</p>
<p>Autosomal dominant polycystic kidney disease (ADPKD) is one of the most prevalent monogenic human diseases. It is mostly caused by pathogenic variants in PKD1 or PKD2 genes that encode interacting transmembrane proteins polycystin-1 (PC1) and polycystin-2 (PC2). Among many pathogenic processes described in ADPKD, those associated with cAMP signaling, inflammation, and metabolic reprogramming appear to regulate the disease manifestations. Tolvaptan, a vasopressin receptor-2 antagonist that regulates cAMP pathway, is the only FDA-approved ADPKD therapeutic. Tolvaptan reduces renal cyst growth and kidney function loss, but it is not tolerated by many patients and is associated with idiosyncratic liver toxicity. Therefore, additional therapeutic options for ADPKD treatment are needed. As drug repurposing of FDA-approved drug candidates can significantly decrease the time and cost associated with traditional drug discovery, we used the computational approach signature reversion to detect inversely related drug response gene expression signatures from the Library of Integrated Network-Based Cellular Signatures (LINCS) database and identified compounds predicted to reverse disease-associated transcriptomic signatures in three publicly available kidney transcriptomic data sets of mouse ADPKD models. We focused on a pre-cystic model for signature reversion, as it was less impacted by confounding secondary disease mechanisms in ADPKD, and then compared the resulting candidates’ target differential expression in the two cystic mouse models. We further prioritized these drug candidates based on their known mechanism of action, FDA status, targets, and by functional enrichment analysis. With this approach, we prioritized 29 unique drug targets differentially expressed in ADPKD cystic models and 16 prioritized drug repurposing candidates that target them, including bromocriptine and mirtazapine. Collectively, these indicate drug targets and repurposing candidates that may effectively treat pre-cystic as well as cystic ADPKD.</p>
<hr>
<p><strong>Nathaniel DeVoss</strong> &ndash; <em>February 2nd, 2023</em><br></p>

  </div>
  <footer>
    <ul class="stats">
  
    
    
      <li class="categories">
        <ul>
          
            
            <li><a class="article-category-link" href="https://lasseigne.org/categories/website">website</a></li>
          
            
            <li><a class="article-category-link" href="https://lasseigne.org/categories/updates">updates</a></li>
          
        </ul>
      </li>
    
  
  
    
    
      <li class="tags">
        <ul>
          
            
            <li><a class="article-category-link" href="https://lasseigne.org/tags/lab">lab</a></li>
          
            
            <li><a class="article-category-link" href="https://lasseigne.org/tags/website">website</a></li>
          
            
            <li><a class="article-category-link" href="https://lasseigne.org/tags/updates">updates</a></li>
          
            
            <li><a class="article-category-link" href="https://lasseigne.org/tags/iscb">ISCB</a></li>
          
        </ul>
      </li>
    
  
</ul>

  </footer>
</article>


<div class="pagination">
  
    <a href="/blog/2022-publications/" class="button"><div class="previous"><div>2022 Publications</div></div></a>
  
  
    <a href="/blog/10-simple-rules/" class="button"><div class="next"><div>10 Simple Rules</div></div></a>
  
</div>


      </main>
      <section id="site-sidebar">
  
    <section id="recent-posts">
      <header>
        <h1>Recent posts</h1>
      </header>
      
      <article class="mini-post">
        <section>
          

        </section>
        <header>
          <h1><a href="/blog/10-simple-rules/">10 Simple Rules</a></h1>
          <time class="published" datetime="">February 16, 2023</time>
        </header>
      </article>
      
      <article class="mini-post">
        <section>
          

        </section>
        <header>
          <h1><a href="/blog/2022-preprints/">2022 Preprints</a></h1>
          <time class="published" datetime="">February 2, 2023</time>
        </header>
      </article>
      
      <article class="mini-post">
        <section>
          

        </section>
        <header>
          <h1><a href="/blog/2022-publications/">2022 Publications</a></h1>
          <time class="published" datetime="">February 2, 2023</time>
        </header>
      </article>
      
      
        <a href="/blog/" class="button">See more</a>
      
    </section>
  

  

  
</section>

      <footer id="site-footer">
  
  <p class="copyright">
    
      &copy; 2023
      
        Lasseigne Lab
      
    . <br>
    Theme: <a href='https://github.com/pacollins/hugo-future-imperfect-slim' target='_blank' rel='noopener'>Hugo Future Imperfect Slim</a><br>A <a href='https://html5up.net/future-imperfect' target='_blank' rel='noopener'>HTML5 UP port</a> | Powered by <a href='https://gohugo.io/' title='0.99.1' target='_blank' rel='noopener'>Hugo</a>
  </p>
</footer>
<a id="back-to-top" href="#" class="fas fa-arrow-up fa-2x"></a>

      <script src="/js/highlight.pack.js"></script>
    <script>hljs.initHighlightingOnLoad();</script><script src="/js/bundle.min.d0b3e0b5f2cfc7467ead7d316ecb9dea4d29ef3c23ad300df9d7017ff98b2331.js" integrity="sha256-0LPgtfLPx0Z&#43;rX0xbsud6k0p7zwjrTAN&#43;dcBf/mLIzE="></script>
    <script src="/js/add-on.js"></script>
    </div>
  </body>
</html>
