---
title: "GTEx Guide"
subtitle: "Version 1.0"
author: 
  - name: "Sasha Thalluri, Lasseigne Lab, CDIB, UAB"
  - name: "Developed with assistance from the Lasseigne Lab and Dr. Lara Ianov, UAB"
    affiliation: For questions, email Dr. Lasseigne at bnp0001@uab.edu
date: "5/27/2020"
output:
  html_document:
    theme: spacelab
    toc: yes
    toc_float:
      collapsed: no
      smooth_control: yes
  pdf_document:
    toc: yes
---

# Introduction into GTEx
***
#### This is an SOP on getting started with Genotype-Tissue Expression (GTEx). Here you can find resources, tools, and workflows that will help get you started with GTEx.
<br>

GTEx is a data resource and tissue bank established to study the relationships between variants and gene expression across several tissue types and different people.

It was originally created to study the relationship between genetic variants and gene expression in multiple human samples across tissues. It has whole genome sequencing, RNA-sequencing, SNP arrays, and whole exome data from 960 deceased adults with multiple different tissue types harvested from those individuals. From each individual, GTEx has obtained around 50 tissue samples. 

The goals of GTEx are to increase understanding of how changes in genes contribute to common human diseases and to identify the pieces of DNA modulating genes. 

<br>
<br>

# Resources 
***
Here, you can find resources that will help in giving an overview of what GTEx is.
<br>


<a href="https://commonfund.nih.gov/gtex" target="_blank">Common Fund's Program Snapshot of GTEx</a>

* An overall introduction to GTEx
* Gives information regarding the entire program

<a href="https://www.nature.com/articles/ng.2653" target="_blank">The Genotype-Tissue Expression (GTEx) project</a>

* The paper that gives an introduction into GTEx

<a href="https://www.gtexportal.org/home/" target="_blank">GTEx Portal</a>

* A user-friendly GUI that allows you to interact with GTEx data
* Includes:
  * Gene expression
  * QTLs
  * Histology images

<a href="https://www.gtexportal.org/home/videos" target="_blank">GTEx Training Videos</a>

* Videos that give an introduction to GTEx data and the portal

<a href="https://gtexportal.org/home/publicationsPage" target="_blank">GTEx Publications Page</a>

* A list of nearly all of GTEx publications

<a href="https://www.nature.com/collections/dcfzxywzby/resources" target="_blank">GTEx Pilot Phase publications </a>

* Pilot Publications on Nature Collections
* All of the manuscripts have been posted until 2017. 

<a href="https://www.gtexportal.org/home/faq" target="_blank">GTEx FAQs</a>

* FAQ page for GTEx
* Answers that could be helpful during your analysis 

<br>
<br>


# Tools
*** 
Here are R packages and tools that can be used with GTEx data. 
<br>
<br>

**UCSCXenaTools**

This is an R Package from UCSC Xena platform, from Cancer Multi-omics to Single-cell RNA-sequencing. The public omics data from UCSC Xena are supported through multiple turn-key Xena Hubs, which are a collection of UCSC-hosted public databases such as TCGA, ICGC, TARGET, GTEx, CCLE, and others. To access this package and the vignette that corresponds to it, click <a href="https://cran.r-project.org/web/packages/UCSCXenaTools/vignettes/USCSXenaTools.html" target="_blank">here.</a>

**recount**

This is an R package that utilizes processed data from TCGA, SRA, and GTEx. Using this package, you can easily access RNA-seq data (e.g.: raw counts) from numerous sources in the R environment to perform downstream analysis with complementary packages (e.g.: DESeq2). Also, you can visualize base-level genome coverage data and perform analyses at multiple feature levels. To access this package, click <a href="https://bioconductor.org/packages/release/bioc/html/recount.html" target="_blank">here.</a> To access the R Shiny app, recount2, click <a href="https://jhubiostatistics.shinyapps.io/recount/" target="_blank">here.</a> 

**TCGAbiolinks**

This is an R package that was originally developed to 
perform integrative analysis with Genomic Data Commons data, including the data from The Cancer Genome Atlas. New functionalities allow for normal samples from the GTEx project to be used with this package.To access the the paper that gives an overview of using GTEx data with TCGAbiolinks, click <a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1006701" target="_blank">here.</a> To access the package, click <a href="https://bioconductor.org/packages/release/bioc/html/TCGAbiolinks.html" target="_blank">here.</a> 

There is a workshop that has an example of using the TCGAbiolinks package to access GTEx data. You can find this example in the link <a href="http://biocworkshops2019.bioconductor.org.s3-website-us-east-1.amazonaws.com/page/Bioc2019PanCancerStudy__Bioc2019_workshop/" target="_blank">here.</a> In particular, look at the section titled **GTEx gene expression data**.

<br>
<br>

# Workflows
*** 
Here are workflows that can be used with GTEx data. 
<br>
<br>

**Working with Big Data in Genomics**

Here is an alternative introduction to working with GTEx data in R. The goal of this workflow is to teach you how to work with large, complex data sets and genomic data in general. To access this workflow, click <a href="https://myweb.uiowa.edu/pbreheny/misc/big-data-lab.html" target="_blank">here.</a>


**GTEx Tutorials**

Here are GTEx tutorials in R. This will help you in selecting samples, loading expression data, and understanding cis-eQTL data. To access the tutorials, click <a href="https://github.com/broadinstitute/gtex-tutorials/blob/master/GTEx_data_tutorial_R.ipynb" target="_blank">here.</a>

* Note: The hashmap package is not working, so that tutorial is obsolete. 


**recountWorkflow**

Here is a workflow for recount. It explains in detail how to use the recount package and how to integrate it with other Bioconductor packages for several analyses that can be carried out with the recount2 resource. To access the workflow, click <a href="https://bioconductor.org/packages/devel/workflows/vignettes/recountWorkflow/inst/doc/recount-workflow.html" target="_blank">here.</a>