---
title: "DNA Methylation Guide"
subtitle: "Version 1.0"
author: 
  - name: "Sasha Thalluri, Lasseigne Lab, CDIB, UAB"
  - name: "Developed with assistance from the Lasseigne Lab and Dr. Lara Ianov, UAB"
    affiliation: For questions, email Dr. Lasseigne at bnp0001@uab.edu
date: "9/15/2020"
output:
  html_document:
    theme: spacelab
    toc: yes
    toc_float:
      collapsed: no
      smooth_control: yes
  pdf_document:
    toc: yes
---

# Introduction into DNA Methylation
***
#### This is an SOP on getting started with DNA Methylation. Here you can find papers, courses, databases, and tools.
<br>
<br> 

# Papers
***
This is a list of links to papers that have essential information on DNA Methylation.
<br>
<br> 

<a href="https://www.ncbi.nlm.nih.gov/pubmed/30100054">Principles of DNA methylation and their implications for biology and medicine.</a>

* Discusses that methylation patterns are established in a programmed process that continues throughout development, thus setting up stable gene expression profiles

<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6147084/">DNA methylation in human diseases</a>

* Discusses findings on DNA methylation in common diseases
* Describes promises and unique challenges of epigenetic epidemiological studies and proposes some potential solutions

<a href="https://academic.oup.com/bfg/article/15/6/432/2453166">Methods for genome-wide DNA methylation analysis in human cancer</a>

* Presents the current key technologies used to detect high-throughput genome-wide DNA methylation, and the available cancer-associated methylation databases
* Focuses on the computational methods for preprocessing, analyzing and interpreting the cancer methylome data
* Summarizes emerging challenges in the computational analysis of cancer methylome data

<a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-014-0408-0">Whole Genome Bisulfite Sequencing</a>

* Presents an integrated analysis of whole-genome bisulfite sequencing and RNA sequencing data from human samples and cell lines
* Their results suggest that DNA methylation outside promoters also plays critical roles in gene regulation

<a href="https://pubmed.ncbi.nlm.nih.gov/19151715/">The human colon cancer methylome shows similar hypo- and hypermethylation at conserved tissue-specific CpG island shores</a>

* Shows that most methylation alterations in colon cancer do not occur in promoters nor CpG islands, but in sequences up to 2 kb distant, which we term 'CpG island shores'
* Discusses that methylation changes in cancer are at sites that vary normally in tissue differentiation, consistent with the epigenetic progenitor model of cancer, which proposes that epigenetic alterations affecting tissue-speciﬁc differentiation are the predominant mechanism by which epigenetic changes cause cancer

<a href="https://cancerres.aacrjournals.org/content/73/19/5858">The CpG Island Methylator Phenotype: What's in a Name?</a>

*  Discusses the use of the term CIMP in cancer, its significance in clinical practice, and future directions that may aid in identifying the true cause and definition of CIMP in different forms of human neoplasia.

<a href="https://www.ncbi.nlm.nih.gov/pubmed/29672679">Disease prediction by cell-free DNA methylation</a>

* Provides thorough reviews and discussions on the statistical method developments and data analysis strategies for using cell-free DNA methylation profiles to construct disease diagnostic models
* Focuses on two important aspects: marker selection and prediction model construction

<a href="https://bmcmedicine.biomedcentral.com/articles/10.1186/s12916-014-0235-x">DNA methylation profiling reveals novel diagnostic biomarkers in renal cell carcinoma</a>

* Identifies a single panel of DNA methylation biomarkers that reliably distinguishes tumor from benign adjacent tissue in all of the most common kidney cancer histologic subtypes and a second panel does the same specifically for clear cell renal cell carcinoma tumors


# Tools
*** 
Here are some R packages and tools that can be used for DNA methylation analysis. 
<br>
<br>

**methylKit**
This is an R package for analysis and annotation of DNA methylation information obtained by high-throughput bisulfite sequencing.The package is designed to deal with sequencing data from RRBS and its variants. But, it can potentially handle whole-genome bisulfite sequencing data if proper input format is provided. To access the R package, click <a href="https://www.bioconductor.org/packages/release/bioc/html/methylKit.html">here.</a> To access the vignette, click <a href="https://bioconductor.org/packages/release/bioc/vignettes/methylKit/inst/doc/methylKit.html">here.</a> 

**IlluminaHumanMethylation450kmanifest**

This is an R package used for annotation for Illumina's 450k methylation arrays. To access the R package, click <a href="https://bioconductor.org/packages/release/data/annotation/html/IlluminaHumanMethylation450kmanifest.html">here.</a>

**minfi**

This is an R package used to analyze Illumina’s Methylation arrays, specifically the 450k and EPIC (also known as the 850k) arrays. This package includes preprocessing, QC assessments, identification of interesting methylation loci and plotting functionality. To access the R package, click <a href="http://bioconductor.org/packages/release/bioc/html/minfi.html">here.</a> To access the vignette, click <a href="http://bioconductor.org/packages/release/bioc/vignettes/minfi/inst/doc/minfi.html">here.</a>

**methyAnalysis**

This is an R package that was developed for chromosome location based DNA methylation analysis and visualization. The package also includes functions of estimating the methylation levels from Methy-Seq data. To access the R package, click <a href="http://bioconductor.org/packages/release/bioc/html/methyAnalysis.html">here.</a>  To access the vignette, click <a href="https://bioconductor.org/packages/release/bioc/vignettes/methyAnalysis/inst/doc/methyAnalysis.pdf">here.</a> 

**A cross-package Bioconductor workflow for analysing methylation array data**

This is a Bioconductor workflow that uses multiple packages for the analysis of methylation array data. This workflow demonstrates steps involved in a typical differential methylation analysis pipeline including: quality control, filtering, normalization, data exploration and statistical testing for probe-wise differential methylation. To access the vignette, click <a href="https://bioconductor.org/packages/release/workflows/vignettes/methylationArrayAnalysis/inst/doc/methylationArrayAnalysis.html">here.</a> 


