<!doctype html>
<html lang="en">
  <head>
  <meta charset="utf-8">
<title>RNA-Seq Guide - Lasseigne Lab</title>
<meta name="viewport" content="width=device-width, initial-scale=1">


  <link rel="apple-touch-icon" sizes="180x180" href="/favicon/apple-touch-icon.png?v=1">
  <link rel="icon" type="image/png" sizes="32x32" href="/favicon/favicon-32x32.png?v=1">
  <link rel="icon" type="image/png" sizes="16x16" href="/favicon/favicon-16x16.png?v=1">
  <link rel="manifest" href="/favicon/site.webmanifest?v=1">
  
    <link rel="mask-icon" href="/favicon/safari-pinned-tab.svg?v=1" color="#603cba">
    <link rel="shortcut icon" href="/favicon/favicon.ico?v=1">
    <meta name="msapplication-config" content="/favicon/browserconfig.xml?v=1">
  
  <meta name="msapplication-TileColor" content="#603cba">
  <meta name="theme-color" content="#5bbad5">

<meta name="generator" content="Hugo 0.99.1" /><meta itemprop="name" content="RNA-Seq Guide">
<meta itemprop="description" content="Introduction into RNA-Seq This is an SOP on getting started with RNA-seq. Here you can find information on resources, aligners, DESeq2, and pathway analysis tools. 
  Resources This is a list of links to resources that have essential information on RNA-seq and the recommended R packages for RNA-seq analysis.
 Introduction to RNA Sequencing  Background lecture for RNA-seq Here are the slides used in the lecture  RNA Sequencing and Analysis  Paper giving an overall introduction to RNA-seq  RNA sequencing data: hitchhiker’s guide to expression analysis  Paper stating the developments in RNA-seq data analysis Focuses on quantification of gene expression and statistical approaches for differential expression  RNA sequencing: the teenage years  Paper stating the different RNA-seq methods, while naming certain RNA technologies and good computational tools for data analysis    About Aligners Contributed by Dr.">

<meta itemprop="wordCount" content="953">
<meta itemprop="keywords" content="" /><meta property="og:title" content="RNA-Seq Guide" />
<meta property="og:description" content="Introduction into RNA-Seq This is an SOP on getting started with RNA-seq. Here you can find information on resources, aligners, DESeq2, and pathway analysis tools. 
  Resources This is a list of links to resources that have essential information on RNA-seq and the recommended R packages for RNA-seq analysis.
 Introduction to RNA Sequencing  Background lecture for RNA-seq Here are the slides used in the lecture  RNA Sequencing and Analysis  Paper giving an overall introduction to RNA-seq  RNA sequencing data: hitchhiker’s guide to expression analysis  Paper stating the developments in RNA-seq data analysis Focuses on quantification of gene expression and statistical approaches for differential expression  RNA sequencing: the teenage years  Paper stating the different RNA-seq methods, while naming certain RNA technologies and good computational tools for data analysis    About Aligners Contributed by Dr." />
<meta property="og:type" content="article" />
<meta property="og:url" content="/education_resources/guides/rmds/rnaseqguide/" /><meta property="article:section" content="education_resources" />



<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="RNA-Seq Guide"/>
<meta name="twitter:description" content="Introduction into RNA-Seq This is an SOP on getting started with RNA-seq. Here you can find information on resources, aligners, DESeq2, and pathway analysis tools. 
  Resources This is a list of links to resources that have essential information on RNA-seq and the recommended R packages for RNA-seq analysis.
 Introduction to RNA Sequencing  Background lecture for RNA-seq Here are the slides used in the lecture  RNA Sequencing and Analysis  Paper giving an overall introduction to RNA-seq  RNA sequencing data: hitchhiker’s guide to expression analysis  Paper stating the developments in RNA-seq data analysis Focuses on quantification of gene expression and statistical approaches for differential expression  RNA sequencing: the teenage years  Paper stating the different RNA-seq methods, while naming certain RNA technologies and good computational tools for data analysis    About Aligners Contributed by Dr."/>
<meta name="twitter:site" content="@bnlasse"/>
<link rel="stylesheet" href="/css/bundle.min.262e62e9c1615dd1ac95d339cfc4ca2167aee72b71d853838920a22e7088b06b.css" integrity="sha256-Ji5i6cFhXdGsldM5z8TKIWeu5ytx2FODiSCiLnCIsGs=">
        <link rel="stylesheet" href="/css/add-on.css">
</head>

  <body>
    
<header id="site-header">
  <nav id="site-nav">
    <h1 class="nav-title">
      <a href="/">
        
          
            education_resources
          
        
      </a>
    </h1>
    <menu id="site-nav-menu" class="flyout-menu">
      
        
          
          
            <a href="/" class="link"><i class='fa fa-home'></i> Home</a>
          
        
      
        
          
          
            <a href="/team/" class="link"><i class='far fa-id-card'></i> Team</a>
          
        
      
        
          
          
            <a href="/research/" class="link"><i class='fas fa-microscope'></i> Research</a>
          
        
      
        
          
          
            <a href="/education/" class="link"><i class='fas fa-chalkboard-teacher'></i> Education</a>
          
        
      
        
          
          
            <a href="/publications/" class="link"><i class='far fa-file-alt'></i> Publications</a>
          
        
      
        
          
          
            <a href="/funding/" class="link"><i class='fas fa-dollar-sign'></i> Funding</a>
          
        
      
        
          
          
            <a href="/contact-us/" class="link"><i class='far fa-envelope'></i> Contact</a>
          
        
      
      
      
    </menu>
    
    
    
    <a href="#site-nav" class="nav-toggle"><i class="fas fa-bars fa-2x"></i></a>
  </nav>
  
  
</header>

    <div id="wrapper">
      <section id="site-intro" >
  <a href="/"><img src="/img/main/lablogo.png" class="circle" width="200" alt="Lasseigne Lab" /></a>
  <header>
    <h1>Lasseigne Lab</h1>
  </header>
  <main>
    <p>Because Science.</p>
  </main>
  
    <footer>
      <ul class="socnet-icons">
        

        <li><a href="//github.com/lasseignelab" target="_blank" rel="noopener" title="GitHub" class="fab fa-github"></a></li>











<li><a href="//linkedin.com/in/brittany-lasseigne-phd" target="_blank" rel="noopener" title="LinkedIn" class="fab fa-linkedin"></a></li>















<li><a href="//twitter.com/bnlasse" target="_blank" rel="noopener" title="Twitter" class="fab fa-twitter"></a></li>







<li><a href="//researchgate.net/profile/Brittany_Lasseigne" target="_blank" rel="noopener" title="Research Gate"><i class="ai ai-researchgate"></i></a></li>


<li><a href="mailto:bnp0001@uab.edu" target="_blank" title="Email" class="far fa-envelope"></a></li>

      </ul>
    </footer>
  
</section>

      <main id="site-main">
        <article class="post">
  <header>
  <div class="title">
    
      <h2><a href="/education_resources/guides/rmds/rnaseqguide/">RNA-Seq Guide</a></h2>
    
    
  </div>
</header>

  <section id="socnet-share">
    





  </section>
  

  <div class="content">
    


<div id="introduction-into-rna-seq" class="section level1">
<h1>Introduction into RNA-Seq</h1>
<hr />
<div id="this-is-an-sop-on-getting-started-with-rna-seq.-here-you-can-find-information-on-resources-aligners-deseq2-and-pathway-analysis-tools." class="section level4">
<h4>This is an SOP on getting started with RNA-seq. Here you can find information on resources, aligners, DESeq2, and pathway analysis tools.</h4>
<p><br></p>
</div>
</div>
<div id="resources" class="section level1">
<h1>Resources</h1>
<hr />
<p>This is a list of links to resources that have essential information on RNA-seq and the recommended R packages for RNA-seq analysis.</p>
<ul>
<li><a href="https://www.youtube.com/watch?v=YzkTc6VNNjo&feature=youtu.be" target="_blank">Introduction to RNA Sequencing</a>
<ul>
<li>Background lecture for RNA-seq</li>
<li><a href="https://drive.google.com/file/d/1Ldp6qrAMy0NVCCAu5cueT4YFq0VArORH/view" target="_blank">Here</a> are the slides used in the lecture</li>
</ul></li>
<li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4863231/" target="_blank">RNA Sequencing and Analysis</a>
<ul>
<li>Paper giving an overall introduction to RNA-seq</li>
</ul></li>
<li><a href="https://peerj.com/preprints/27283/" target="_blank">RNA sequencing data: hitchhiker’s guide to expression analysis</a>
<ul>
<li>Paper stating the developments in RNA-seq data analysis</li>
<li>Focuses on quantification of gene expression and statistical approaches for differential expression</li>
</ul></li>
<li><a href="https://www.nature.com/articles/s41576-019-0150-2" target="_blank">RNA sequencing: the teenage years</a>
<ul>
<li>Paper stating the different RNA-seq methods, while naming certain RNA technologies and good computational tools for data analysis</li>
</ul></li>
</ul>
</div>
<div id="about-aligners" class="section level1">
<h1>About Aligners</h1>
<hr />
<p class="text-muted">
Contributed by Dr. Lara Ianov, UAB
</p>
<p>When RNA-seq alignment and mapping are discussed, there are generally two primary methods of alignment: <strong>standard splice-aware alignment to the genome</strong> (e.g.: STAR, HISAT2) and <strong>quasi-mapping/pseudoalignment approaches</strong>, which utilize k-mer based counting methods to map fragments to the transcriptome (e.g.: Salmon, kallisto). STAR <a href="https://academic.oup.com/bioinformatics/article/29/1/15/272537" target="_blank">(Dobin et al., 2013)</a> is usually my choice for a splice-aware aligner and Salmon <a href="https://pubmed.ncbi.nlm.nih.gov/28263959/" target="_blank">(Patro, Duggal, Love, Irizarry, &amp; Kingsford, 2017)</a> for quasi-mapping. <em>The choice between the two generally depend on the specific long-term goal of a study.</em> It is worth noting that there are significant differences in the computational efficiency among these methods, but this goes beyond the scope of this synopsis.</p>
<p>While some benchmarking studies or computational community-based benchmarking have pointed to increased performance from quasi-mapping/pseudoalignment approaches, others have indicated that all modern methods perform similarly for mRNAs and highly abundant genes <a href="https://cgatoxford.wordpress.com/2016/08/17/why-you-should-stop-using-featurecounts-htseq-or-cufflinks2-and-start-using-kallisto-salmon-or-sailfish/" target="_blank">(Smith, 2016;</a> <a href="https://link.springer.com/article/10.1186/s12864-018-4869-5" target="_blank">Wu, Yao, Ho, Lambowitz, &amp; Wilke, 2018)</a>. Thus, one may conclude that if the goal of the study is to perform standard mRNA-seq, any differences found among the methods will not result in a large loss of information if any.</p>
<p>However, beyond standard gene-level mRNA-seq quantification, here is an overview of some of the key points where choosing one approach over another may have a large effect in the interpretation of the results:</p>
<ul>
<li>For long non-coding RNAs quantification, quasi-mapping/pseudoalignment have consistently outperformed standard splice-aware alignment methods <a href="https://academic.oup.com/gigascience/article/8/12/giz145/5663671" target="_blank">(Zheng, Brennan, Hernaez, &amp; Gevaert, 2019)</a>.</li>
<li>On the other hand, it has been reported that standard splice-aware RNA-seq aligners are more sensitive towards small RNAs (including miRNAs) or low abundant genes <a href="https://link.springer.com/article/10.1186/s12864-018-4869-5" target="_blank">(Wu et al., 2018)</a>.</li>
<li>Gene-level vs transcript-levels quantification: it is important to keep in mind that in most cases standard splice-aware aligners and down-stream quantification methods are meant to target gene-level analysis as there are challenges with multi-mapping reads for transcript variant quantification. Transcript-level analysis is a much-debated topic, but the use of quasi-mapping coupled with novel down-stream quantification tools <a href="https://www.biorxiv.org/content/10.1101/2020.04.07.029967v1.abstract" target="_blank">(Sarkar, Srivastava, Bravo, Love, &amp; Patro, 2020)</a> for transcript-level analysis are more sensitive <a href="https://cgatoxford.wordpress.com/2016/08/17/why-you-should-stop-using-featurecounts-htseq-or-cufflinks2-and-start-using-kallisto-salmon-or-sailfish/" target="_blank">(Smith, 2016)</a>.
<ul>
<li>However, if the goal of the analysis is to investigate splicing events (e.g.: classifying what type of splicing event occurred) or fusion events rather than transcript abundance quantification, most of the tools developed currently depend on outputs from standard splice-aware RNA-seq aligners (although this may change in the near future as tools such as Terminus cited above finish development/are updated).</li>
</ul></li>
<li>If variant analysis is to be performed on RNA-seq, then splice-aware alignment methods must be used. It could also be argued that if RNA-seq data will be closely compared to whole-genome sequencing dataset, then it may be best to use the slice-aware aligner methods as well (especially if genome browsers will be used heavily).</li>
<li>For diagnostic goals (e.g.: rare disease cases), most tools available have a dependency on standard alignment approaches as well.</li>
</ul>
</div>
<div id="deseq2" class="section level1">
<h1>DESeq2</h1>
<hr />
<p>DESeq2 is an R package used for differential analysis on count data from high-throughput sequencing assays. It allows for quantitative analysis focused on strength rather than the mere presence of differential expression.</p>
<p><strong>The required input for DESeq2 is gene or transcript counts.</strong></p>
<p><a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-014-0550-8" target="_blank">Here</a> is the paper that will give an overall introduction into DESeq2. It explains how DESeq2 uses shrinkage estimation for dispersions and fold changes to improve stability and interpretability of estimates.</p>
<p>Instructions on how to install DESeq2 into RStudio can be found <a href="http://bioconductor.org/packages/release/bioc/html/DESeq2.html" target="_blank">here.</a></p>
<p>After installing DESeq2, you can begin the vignette. The link to the vignette can be found <a href="http://bioconductor.org/packages/release/bioc/vignettes/DESeq2/inst/doc/DESeq2.html#standard-workflow" target="_blank">here.</a></p>
<p><a href="http://master.bioconductor.org/packages/release/workflows/vignettes/rnaseqGene/inst/doc/rnaseqGene.html" target="_blank">Here</a> is Mike Love’s guide for DESeq2. This is an additional resource that explains how to use the package and demonstrates useful workflows.</p>
<p>Please be aware there are many other approaches for estimating differential expression that may be appropriate, or more appropriate (e.g., time course analysis), for your project. We recommend discussing with a knowledgeable data scientist if you are unsure if DESeq2 is a good option for your experimental design.</p>
</div>
<div id="pathway-analysis-tools-for-r" class="section level1">
<h1>Pathway Analysis Tools for R</h1>
<hr />
<p>Below are links to vignettes/tutorials for R packages that are helpful for pathway analysis. There are many other tools and analytical approaches and we intend to add to this section in the future.</p>
<p><strong>gprofiler2</strong></p>
<p>The gprofiler2 package provides a tool set for functional enrichment analysis and visualization.
It is primarily used to visualize gene lists, convert gene/protein/SNP identifiers to numerous namespaces, and map orthologous genes across species. <a href="https://www.nature.com/articles/s41596-018-0103-9" target="_blank">Here</a> is the paper that will give an overall introduction to pathway analysis using the gprofiler2 package.</p>
<p><a href="https://cran.r-project.org/web/packages/gprofiler2/vignettes/gprofiler2.html" target="_blank">Here</a> is the link to the vignette, which will give you clear instructions on how to use the package.</p>
<p><strong>GAGE</strong></p>
<p>Generally Applicable Gene-set Enrichment (GAGE) is a method for gene set or pathway analysis. The gage package can be used on microarray or RNA-seq data for routine and advanced gene set analyses. <a href="https://bioconductor.org/packages/release/bioc/html/gage.html" target="_blank">Here</a> is the link to the Bioconductor webpage, which will background information on the package and instructions on how to install it into RStudio.</p>
<a href="https://bioconductor.org/packages/release/bioc/vignettes/gage/inst/doc/gage.pdf" target="_blank">Here</a> is the link to the vignette.
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<center>
Version 1.1: Added RNA-Seq alignment information from Dr. Ianov
</center>
<p><br></p>
</div>

  </div>
  <footer>
    <ul class="stats">
  
    <li class="categories">
      <ul>
        <li>None</li>
      </ul>
    </li>
  
  
    <li class="tags">
      <ul>
        <li>None</li>
      </ul>
    </li>
  
</ul>

  </footer>
</article>


<div class="pagination">
  
    <a href="/education_resources/guides/rmds/singlecellguide/" class="button"><div class="previous"><div>Single-Cell Transcriptomics Guide</div></div></a>
  
  
    <a href="/education_resources/guides/rmds/reproducibleresearchguide/" class="button"><div class="next"><div>Reproducible Research Data Science Guide</div></div></a>
  
</div>


      </main>
      <section id="site-sidebar">
  
    <section id="recent-posts">
      <header>
        <h1>Recent posts</h1>
      </header>
      
      <article class="mini-post">
        <section>
          

        </section>
        <header>
          <h1><a href="/blog/second-update/">Second Update</a></h1>
          <time class="published" datetime="">August 1, 2022</time>
        </header>
      </article>
      
      <article class="mini-post">
        <section>
          

        </section>
        <header>
          <h1><a href="/blog/first-update/">First Update</a></h1>
          <time class="published" datetime="">November 2, 2020</time>
        </header>
      </article>
      
      <article class="mini-post">
        <section>
          

        </section>
        <header>
          <h1><a href="/blog/new-website/">New Website!</a></h1>
          <time class="published" datetime="">March 27, 2020</time>
        </header>
      </article>
      
      
        <a href="/blog/" class="button">See more</a>
      
    </section>
  

  

  
</section>

      <footer id="site-footer">
  
  <p class="copyright">
    
      &copy; 2022
      
        Lasseigne Lab
      
    . <br>
    Theme: <a href='https://github.com/pacollins/hugo-future-imperfect-slim' target='_blank' rel='noopener'>Hugo Future Imperfect Slim</a><br>A <a href='https://html5up.net/future-imperfect' target='_blank' rel='noopener'>HTML5 UP port</a> | Powered by <a href='https://gohugo.io/' title='0.99.1' target='_blank' rel='noopener'>Hugo</a>
  </p>
</footer>
<a id="back-to-top" href="#" class="fas fa-arrow-up fa-2x"></a>

      <script src="/js/highlight.pack.js"></script>
    <script>hljs.initHighlightingOnLoad();</script><script src="/js/bundle.min.d0b3e0b5f2cfc7467ead7d316ecb9dea4d29ef3c23ad300df9d7017ff98b2331.js" integrity="sha256-0LPgtfLPx0Z&#43;rX0xbsud6k0p7zwjrTAN&#43;dcBf/mLIzE="></script>
    <script src="/js/add-on.js"></script>
    </div>
  </body>
</html>
